modules/social_features/social_profile/modules/social_profile_privacy/social_profile_privacy.module

```diff
@@ -310,6 +310,7 @@ function _social_profile_privacy_fields_submit($form, FormStateInterface $form_s
 
   $profiles = \Drupal::entityQuery('profile')
     ->condition('uid', $uid)
+    ->accessCheck(FALSE)
```

Comment:
Shouldn't this be TRUE? It's a form submit where it's triggered by the user himself.

Although I guess even if you can manipulate the post the only result is that you can force a cache invalidate for other profiles.

Reply:
Seems like that wouldn't hurt, eventhough I thought it might make sense that a user actually saving the user entity might not have access to the user profile for certain cases I don't think those ever we're part of our use case (as the default is TRUE and that has been running for a while now)